resources/assets/js/components/Conference.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="conference">
+        <friends-list :list="newFriends" key="new-friends" :conference-id="conferenceId" descriptor="New"></friends-list>
+
+        <friends-list :list="onlineFriends" key="online-friends" :conference-id="conferenceId" descriptor="Online"></friends-list>
+    </div>
+</template>
+
+<script>
+    export default {
+        data: function() {
+            return {
+                currentUserId: Confomo.userId,
+                conferenceId: Confomo.conferenceId,
+
+                newFriends: [],
+                onlineFriends: []
+            };
+        },
+
+        components: {
+            FriendsList: require('./FriendsList.vue')
```

Comment:
@mattstauffer would it make sense to import the modules directly within the component itself? That way each component would explicitly define it's dependencies and we'd have the benefit of a consistent notation. 

```javascript
<script>
    import FriendsList from './FriendsList.vue';

    export default {
        components: { FriendsList }
    };
</script>
```